digitalidentity/share_session_notification_builder.go

```diff
@@ -49,10 +49,10 @@ func (b *ShareSessionNotificationBuilder) Build() (ShareSessionNotification, err
 // MarshalJSON returns the JSON encoding
 func (a *ShareSessionNotification) MarshalJSON() ([]byte, error) {
 	return json.Marshal(&struct {
-		Url       string              `json:"url"`
-		Method    string              `json:"method"`
-		VerifyTls *bool               `json:"verifyTls"`
-		Headers   map[string][]string `json:"headers"`
+		Url       string               `json:"url"`
+		Method    *string              `json:"method,omitempty"`
+		VerifyTls *bool                `json:"verifyTls,omitempty"`
+		Headers   *map[string][]string `json:"headers,omitempty"`
```

Comment:
```suggestion
		Headers   map[string][]string `json:"headers,omitempty"`
```

does not need to be pointer to map, maps are inherently pointers and can be nullable already